packages/cli/src/Server.ts

```diff
@@ -169,8 +163,8 @@ import type {
 import { DEFAULT_EXECUTIONS_GET_ALL_LIMIT, validateEntity } from './GenericHelpers';
 import { ExecutionEntity } from './databases/entities/ExecutionEntity';
 import { SharedWorkflow } from './databases/entities/SharedWorkflow';
-import { SharedCredentials } from './databases/entities/SharedCredentials';
 import { RESPONSE_ERROR_MESSAGES } from './constants';
+import { credentialsEndpoints } from './endpoints/credentials.endpoints';
```

Comment:
Renamed as suggested, but please consider:
- File searchability. This is a monorepo where we already have four different `credentials.ts` files that match on file search - even searching for `credentials name` (file plus part of new dirname) actually brings up `CredentialsSelectModal.vue` first, so an even a longer search string ends up being needed to find this. On the other hand, using a dotted suffix like `.endpoints` brings up the file already on `credentials.e`, followed also by the relevant tests `credentials.endpoints.test.ts`, with all other similar but irrelevant `credentials` matches getting filtered out. We could use `.api` instead of `.endpoints` to keep it concise, ideally with mirrored in the tests.
- Consistency. I think this bundle of endpoints does not constitute a namespace (e.g. thinking of wildcard `*` imports, TS types grouped in namespaces, etc.) because  in `Server.ts` we are not referencing them as `<namespace>.<entity>`. These endpoints are not namespaced, just split off into a separate file.

Reply:
Also let's move UM endpoints to this dir as well? In this PR or in another.